app/src/sharedTest/java/org/oppia/app/utility/TabMatcher.kt

```diff
@@ -0,0 +1,50 @@
+package org.oppia.app.utility
+
+import android.view.View
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+import androidx.test.espresso.PerformException
+import com.google.android.material.tabs.TabLayout
+import org.hamcrest.Matcher
+
+/** This class mainly provides a custom matcher to test whether the current tab title is correctly selected in TabLayout. */
+class TabMatcher {
```

Comment:
As informed earlier you were suppose to add the link from where you are using this code. Please add links if you are taking some code snippet from internet.

Reply:
Okay